tests/unit/plugins/v1/test_colcon.py

```diff
@@ -70,12 +71,13 @@ class props:
         self.pip_mock.return_value.list.return_value = {}
 
     def assert_rosdep_setup(
-        self, rosdistro, package_path, rosdep_path, ubuntu_distro, base
+        self, rosdistro, rosversion, package_path, rosdep_path, ubuntu_distro, base
```

Comment:
nitpick: should probably be `ros_version` for consistency with the rest, even if `rosdistro` is off

Reply:
Fixed in 2799f6f.